contracts/chain-adapters/ZkStack_Adapter.sol

```diff
@@ -139,6 +135,7 @@ contract ZkStack_Adapter is AdapterInterface {
         // A bypass proxy seems to no longer be needed to avoid deposit limits. The tracking of these limits seems to be deprecated.
         // See: https://github.com/matter-labs/era-contracts/blob/bce4b2d0f34bd87f1aaadd291772935afb1c3bd6/l1-contracts/contracts/bridge/L1ERC20Bridge.sol#L54-L55
         uint256 txBaseCost = _computeETHTxCost(L2_GAS_LIMIT);
+        address sharedBridge = BRIDGE_HUB.sharedBridge();
```

Comment:
nit: can move this into the `else` statement on line 158 because we don't need it in other branches